react/__tests__/components/Newsletter.test.js

```diff
@@ -69,24 +65,21 @@ test('should handle mutation error', async () => {
         query: subscribeNewsletter,
         variables: { email },
       },
-      error: new Error('ops')
-    }
+      error: new Error('ops'),
+    },
   ]
 
-  const { getByLabelText, getByText } = render(
-    <Newsletter />,
-    { graphql: { mocks, addTypename: false }}
-  )
+  const { getByLabelText, getByText } = render(<Newsletter />, {
+    graphql: { mocks, addTypename: false },
+  })
 
   const input = getByLabelText(/subscribe to our newsletter/i)
   const submit = getByText(/sign up/i)
 
   fireEvent.change(input, { target: { value: email } })
   fireEvent.click(submit)
 
-  await wait()
-
-  const error = getByText(/something went wrong/i)
+  const error = await waitForElement(() => getByText(/something went wrong/i))
 
   expect(error).toBeTruthy()
```

Comment:
It good test not null too, in this case, if the error is a null also passes in the test.

Reply:
Actually null isn't a truthy value.

Reply:
I confused with `toBeDefined` sorry.

Reply:
I'm just fixing the breaking test in this, not creating new ones. I can do it on another PR though